.pipelines/pr-e2e.yaml

```diff
@@ -89,6 +89,16 @@ jobs:
     runSSHTests: true
 
 - template: e2e-job-template.yaml
+  parameters:
+    name: 'k8s_1_21_docker_e2e'
+    k8sRelease: '1.21'
+    apimodel: 'examples/e2e-tests/kubernetes/release/default/definition.json'
+    createVNET: true
+    enableKMSEncryption: false
+    containerRuntime: 'docker'
+    runSSHTests: true
+
+    - template: e2e-job-template.yaml
```

Comment:
I think this extra indentation is too cute for yaml

Reply:
D'oh, thanks for catching that! 👍🏼